src/main/java/de/tum/in/www1/artemis/service/connectors/athena/AthenaFeedbackSendingService.java

```diff
@@ -0,0 +1,122 @@
+package de.tum.in.www1.artemis.service.connectors.athena;
+
+import java.util.List;
+
+import javax.validation.constraints.NotNull;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Profile;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.RestTemplate;
+
+import de.tum.in.www1.artemis.domain.Feedback;
+import de.tum.in.www1.artemis.domain.TextExercise;
+import de.tum.in.www1.artemis.domain.TextSubmission;
+import de.tum.in.www1.artemis.exception.NetworkingError;
+import de.tum.in.www1.artemis.repository.TextBlockRepository;
+import de.tum.in.www1.artemis.service.dto.athena.TextExerciseDTO;
+import de.tum.in.www1.artemis.service.dto.athena.TextFeedbackDTO;
+import de.tum.in.www1.artemis.service.dto.athena.TextSubmissionDTO;
+
+/**
+ * Service for sending feedback to the Athena service for further processing
+ * so that Athena can later give feedback suggestions on new submissions.
+ */
+@Service
+@Profile("athena")
+public class AthenaFeedbackSendingService {
```

Comment:
What do you think of `AthenaFeedbackPublishingService`?

Reply:
I equally like that name, but then I would have to refactor Athena a bit as well (which is fine if there's a real advantage in the new name). I'll ask @FelixTJDietrich and @maximiliansoelch about their opinions as well and then come back to this.

Reply:
I think there is not much of a difference between both. I left another comment asking about feedback changes and deletions, we probably also have to handle those. In that case, Athena would subscribe to feedback and then the publishing name would make more sense to me.

Reply:
I briefly changed this but then reverted the change because I think the verb `send` more intuitively captures that the service only performs an HTTP request. Athena does not really subscribe to anything; Artemis actively sends the submissions and the feedback to Athena.